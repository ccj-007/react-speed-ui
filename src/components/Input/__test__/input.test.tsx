import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Input } from '../../index';

describe('test Input component', () => {
  it('should render', () => {
    const { asFragment } = render(<Input />);
    expect(asFragment()).toMatchSnapshot();
  });
});
