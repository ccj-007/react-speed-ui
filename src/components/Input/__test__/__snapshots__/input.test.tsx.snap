// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Input component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-input-warp"
  >
    <input
      class="speed-input"
      data-testid="input"
      style="padding-right: 30px;"
      value=""
    />
  </div>
</DocumentFragment>
`;
